### PR TITLE
fix(helm): add inheritedParameterContexts option to NiFi ParameterCo…

### PR DESCRIPTION
**Pull request by Roger1uphealth**
| Q               | A
| --------------- | ---
| Bug fix?        | yes
| New feature?    | no
| API breaks?     | no
| Deprecations?   | no
| Related tickets |partially #165
| License         | Apache 2.0

### What's in this PR?

This change will add the option to include inheritedParameterContexts in nifi-parameter-context helm template

### Why?

Inherited parameter contexts were not being deployed as expected due to the missing field in the Helm template.

### Checklist

- [x] Implementation tested
- [x] Error handling code meets the [guideline](docs/error-handling-guide.md)
- [x] Logging code meets the guideline
- [x] User guide and development docs updated (if needed)
- [x] Append changelog with changes

### To Do

- [ ] If the PR is not complete but you want to discuss the approach, list what remains to be done here

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -768,3 +768,4 @@
 ### Removed
 
 ### Fixed Bugs
+ - [PR #476](https://github.com/konpyutaika/nifikop/pull/475) - **[Helm chart]** Added option in nifi-parameter-context helm chart to support setting `inheritedParameterContexts`.
```
Comment (juldrixx) [inline CHANGELOG.md:771]:
```suggestion
### Fixed Bugs

 - [PR #476](https://github.com/konpyutaika/nifikop/pull/475) - **[Helm chart]** Added option in nifi-parameter-context helm chart to support setting `inheritedParameterContexts`.
```

**Review by juldrixx — CHANGES_REQUESTED** [review]

**Comment by juldrixx** [comment]
And can you regenerte the Helm doc using [helm docs](https://github.com/norwoodj/helm-docs)?

```shell
cd helm/nifi-cluster
helm-docs
```

**Review by juldrixx — CHANGES_REQUESTED** [review]
Can you move it to `### Added` instead of `### Fixed Bugs`.

And put a capital letter on `Chart` in `[Helm chart]`.

**Review by juldrixx — CHANGES_REQUESTED** [review]
Can you add it in the `Unreleased` section at the beginning of the file?

**Review by juldrixx — CHANGES_REQUESTED** [review]
Can you add the field in `values.yaml` too with a comment that will be part of the generated documentation like it's done for `secretRefs` and `parameters`.

**Comment by Roger1uphealth** [comment]
@juldrixx thanks for quick review and patience.  I think it's in a spot where it can be reviewed again when you have time.

**Comment by juldrixx** [comment]
Thanks for your contribution🙌.

**Pull request merged**